lib/minimagick/tool.rb

```diff
@@ -0,0 +1,16 @@
+# Hack to monkey-patch MiniMagick to always add the 'quiet'
+# option to every imagemagick command line.
+class MiniMagick::Tool
+  class_attribute :quiet_arg
+  self.quiet_arg = false
+
+  prepend(Module.new do
+    def command
+      if quiet_arg
+        [*executable, *(['-quiet'] + args)]
+      else
+        super
+      end
+    end
+  end)
+end
```

Comment:
I'm not sure if this will succesfully override, or end up replacing the entire `MiniMagick::Tool` class with this new implementation. Maybe we should call it "tool_override.rb", and just explicitly require it, maybe globally in config/application.rb or something?  I am not sure, thoughts?